software/base/src/main/java/brooklyn/entity/brooklynnode/EntityHttpClient.java

```diff
@@ -23,10 +23,29 @@
 import brooklyn.util.http.HttpTool;
 import brooklyn.util.http.HttpToolResponse;
 
+import com.google.common.base.Predicate;
+import com.google.common.base.Predicates;
+import com.google.common.collect.Range;
+
 /**
  * Helpful methods for making HTTP requests to {@link BrooklynNode} entities.
  */
 public interface EntityHttpClient {
+    interface ResponseCodePredicates {
+        static class ResponseCodeHealthyPredicate implements Predicate<Integer> {
+            @Override
+            public boolean apply(Integer input) {
+                return HttpTool.isStatusCodeHealthy(input);
+            }
+        }
+        Predicate<Integer> INFORMATIONAL = Range.closed(100, 199);
```

Comment:
I have a personal preference for methods rather than static constants, which gives us future flexibility. That follows the guava convention, such as the method `Predicates.alwaysTrue()`. In this case I can't see why we'd ever change it, but I'd prefer us sticking to the general guava pattern.